src/Snakemake/workflow/Twist_DNA_workflow.smk

```diff
@@ -1,13 +1,12 @@
 if "units" in config:
     units = pd.read_table(config["units"], index_col=["sample", "unit"], dtype=str)
     units.index = units.index.set_levels([i.astype(str) for i in units.index.levels])  # enforce str in index
-
-
-demultiplex_output = "fastq_temp"
+else:
+    demultiplex_output = "fastq_temp"
+    include: "../rules/Fastq/demultiplex.smk"
 
 
 include: "../rules/Alignment/index_bam.smk"
-include: "../rules/Fastq/demultiplex.smk"
 
 
 if config.get("move_umi", True):
```

Comment:
Varför heter den move_umi och inte move_fastq? Är det för att fastq-filerna måste vara UMI-fixade?

Reply:
Lägger in issue på att byta namn